### PR TITLE
go.mod: update cloud.google.com/go to v0.35.0

### PR DESCRIPTION
**Pull request by jba**

### CODE REVIEW

**Comment by jba** [comment]
Who knows? I literally just changed the `4` to a `5`, then ran `go test ./...`.

Presumably `cloud.google.com/go` no longer has certain dependencies that it used to have.

File: go.sum [hunk 1]
```diff
@@ -90,72 +81,57 @@ github.com/benbjohnson/tmpl v1.0.0/go.mod h1:igT620JFIi44B6awvU9IsDhR77IXWtFigTL
...
 github.com/circonus-labs/circonusllhist v0.1.3/go.mod h1:kMXHVDlOchFAehlya5ePtbp5jckzBHf4XRpQvBOLI+I=
 github.com/client9/misspell v0.3.4/go.mod h1:qj6jICC3Q7zFZvVWo7KLAzC3yx5G7kyvSDkc90ppPyw=
 github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd/go.mod h1:sE/e/2PUdi/liOCUjSTXgM1o87ZssimdTWN964YiIeI=
 github.com/containerd/continuity v0.0.0-20181027224239-bea7585dbfac/go.mod h1:GL3xCUCBDV3CZiTSEKksMWbLE66hEyuu9qyDOOqM47Y=
-github.com/containerd/continuity v0.0.0-20181203112020-004b46473808 h1:4BX8f882bXEDKfWIf0wa8HRvpnBoPszJJXL+TVbBw4M=
 github.com/containerd/continuity v0.0.0-20181203112020-004b46473808/go.mod h1:GL3xCUCBDV3CZiTSEKksMWbLE66hEyuu9qyDOOqM47Y=
 github.com/coreos/bbolt v1.3.1-coreos.6 h1:uTXKg9gY70s9jMAKdfljFQcuh4e/BXOM+V+d00KFj3A=
 github.com/coreos/bbolt v1.3.1-coreos.6/go.mod h1:iRUV2dpdMOn7Bo10OQBFzIJO9kkE559Wcmn+qkEiiKk=
-github.com/coreos/etcd v3.3.11+incompatible h1:0gCnqKsq7XxMi69JsnbmMc1o+RJH3XH64sV9aiTTYko=
+github.com/coreos/etcd v3.3.11+incompatible h1:U0wJghY374q+UrjOM2mfROHSwEspsQVkCACB1PGka1g=
```
Comment (vangent) [inline go.sum]: This one looks sketchy. @eliben just went through a long and painful process of updating dependencies for which the hash had changed, but my understanding was that that was supposed to be a one-time thing due to a backwards-incompatible change in `go`. Which version of `go` are you using?

Comment (jba) [reply go.sum]: The one on my workstation, which is a prerelease of Go 1.12:
```
go version devel +56c9f8e8cf linux/amd64 
```

Comment (vangent) [reply go.sum]: Hmm. Trying cleaning your module cache, reverting `go.sum`, and trying `go test` again?

Comment (eliben) [reply go.sum]: FWIW I have the same go version

Comment (eliben) [reply go.sum]: I just tried running `go mod tidy` in a clean clone and I also get all these many packages removed, without changing anything

Comment (eliben) [reply go.sum]: ... but I don't see hash changes in go.sum

Comment (jba) [reply go.sum]: Did
```
go clean -modcache
git co master -- go.sum
go test ./...
```

**Comment by jba** [comment]
@vangent Any concerns?

**Pull request merged**